tests/console/consoletest_setup.pm

```diff
@@ -47,11 +47,13 @@ sub run {
     select_console 'user-console';
     # Shell enviromental variable MAILCHECK has to be updated for both users
     disable_bash_mail_notification;
+    assert_screen 'here-2fail';
 }
 
 sub post_fail_hook {
     my $self = shift;
-
+    select_console('log-console');
+    $self->SUPER::post_fail_hook();
```

Comment:
I think the parent class method should not be invoked here:

```perl
# lib/consoletest.pm
sub post_fail_hook {
    my ($self) = shift;
    select_console('log-console');
    $self->SUPER::post_fail_hook;
    $self->remount_tmp_if_ro;
    $self->export_logs_basic;
    # Export extra log after failure for further check gdm issue 1127317, also poo#45236 used for tracking action on Openqa
    $self->export_logs_desktop;
}
```

```suggestion
```

Reply:
yes, got updated.